lib/Doctrine/Persistence/ManagerRegistry.php

```diff
@@ -65,7 +65,8 @@ public function getAliasNamespace($alias);
     /**
      * Gets all object manager names.
      *
-     * @return string[] An array of object manager names.
+     * @return array<string,string> An array with object manager names as keys,
+     *                              and service ids as values.
```

Comment:
What's a "service id"? It's the first time the concept of a service appears on that contract.

Reply:
I think we should add a short paragraph to the doc block of this method, about what a service is in our context. Right now, the sudden mention of service IDs is a bit confusing.